src/views/BioPage.jsx

```diff
@@ -1,34 +1,95 @@
+// LIBRARY IMPORTS
 import React from 'react';
 import { useParams } from 'react-router-dom';
+// LOCAL IMPORTS
 import { bioData } from '../assets/bioData.js';
+import SingleCard from '../components/SingleCard.jsx';
+import ResourceCard from '../components/ResourceCard.jsx';
 
 export const BioPage = () => {
   const { id } = useParams();
-  const data = bioData.find(item => item.id === parseInt(id, 10));
+  const currentIndex = bioData.findIndex(item => item.id === parseInt(id, 10));
+
+  const nextCards = bioData
+    .filter(
+      (item, index) =>
+        index !== currentIndex &&
+        index > currentIndex &&
+        index < currentIndex + 4,
+    )
+    .map(item => (
+      <SingleCard
+        key={item.id}
+        id={item.id}
+        image={item.image}
+        name={item.fullName}
+        body={item.occupation}
+      />
+    ));
+
+  const data = bioData[currentIndex];
```

Comment:
It's good to use descriptive names for variables such as 'person' instead of general names like 'data'. The names of variables should be self-explanatory and describe the stored value.

Reply:
Ditto to Razana's comment - `bio` or `person` could be alternative names instead (especially bc `data` is a variable set in `App.jsx` that we use throughout the app)